src/OnlySpans.PolyLeads.Api/Data/Options/LLMOptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using OnlySpans.PolyLeads.Api.Data.Abstractions.Options;
+
+namespace OnlySpans.PolyLeads.Api.Data.Options;
+
+public sealed record LLMOptions : IApplicationOptions
+{
+    public static string Section { get; } = "LLM";
+
+    public string ApiKey { get; init;} = string.Empty;
```

Comment:
```suggestion
    public string ApiKey { get; init; } = string.Empty;
```

в остальных местах также